library/platform_util.c

```diff
@@ -33,11 +33,35 @@
 #include "mbedtls/threading.h"
 
 #include <stddef.h>
+
+#ifndef __STDC_WANT_LIB_EXT1__
+#define __STDC_WANT_LIB_EXT1__ 1
+#endif
 #include <string.h>
 
+#if defined(_WIN32)
+#include <Windows.h>
+#endif
+
+// Detect platforms known to support explicit_bzero()
+#if defined(__GLIBC__) && (__GLIBC__ >= 2) && (__GLIBC_MINOR__ >= 25)
```

Comment:
Doesn't this depend on the `.so` that's loaded into our process at run time? There's `gnu_get_libc_version()`

Reply:
You'd still need to compile against a glibc that supports it though, to get the declaration? In which case it's fair to assume that you run against a glibc that's at least as new as the one you built against?